frontend/src/components/HomePage.js

```diff
@@ -106,19 +123,38 @@ function HomePage() {
                 </DialogActions>
             </Dialog>
 
-            <Dialog open={isDeleteErrorDialogOpen} onClose={() => setDeleteErrorDialogOpen(false)}>
+            <Dialog open={isWrongPasswordDialogOpen} onClose={() => setWrongPasswordDialogOpen(false)}>
                 <DialogTitle>Wrong Password</DialogTitle>
                 <DialogActions>
-                    <Button onClick={() => setDeleteErrorDialogOpen(false)}>Close</Button>
+                    <Button onClick={() => setWrongPasswordDialogOpen(false)}>Close</Button>
                 </DialogActions>
             </Dialog>
 
-            <Dialog open={isDeleteSuccessDialogOpen} onClose={() => setDeleteErrorDialogOpen(false)}>
+            <Dialog open={isDeleteSuccessDialogOpen} onClose={() => setDeleteSuccessDialogOpen(false)}>

```

Comment:
perhaps in the next milestone we can change the behaviour -- when user enters wrong password, it will display wrong password, and ask the user to try again (instead of closing the entire dialog box)